lib/dao/api_service_dao.es6.js

```diff
@@ -62,6 +62,8 @@ foam.CLASS({
         // Change delegate to client once it has finished caching in worker.
         clientDAO.limit(1).select().then(() => {
           this.delegate = clientDAO;
+        }, (err) => {
+          this.warn(`Caching all ${this.of.id} data failed. Using REST API instead. Reason: ${err}`);
```

Comment:
This class should `import: ['warn']`. If it's already working, then it is by an accident of inheritance that we shouldn't rely on.

Reply:
Alright, I removed it after a warning about another class, but that didn't remove the warning so I'll fix that separately.